src/renderer/components/test-utils/get-application-builder.tsx

```diff
@@ -179,6 +182,15 @@ export const getApplicationBuilder = () => {
 
   overrideFsWithFakes(mainDi);
 
+  // Set up ~/.kube as existing as a folder
+  {
+    const { ensureDirSync } = mainDi.inject(fsInjectable);
+    const joinPaths = mainDi.inject(joinPathsInjectable);
+    const homeDirectoryPath = mainDi.inject(homeDirectoryPathInjectable);
+
+    ensureDirSync(joinPaths(homeDirectoryPath, ".kube"));
```

Comment:
Does this access actual filesystem or the fake one? The actual one is not to be accessed in a unit test.

Reply:
This accesses the fake one because we have already overridden `fsInjectable` for `mainDi` above on line 183